File: P5-NascarSeasonWebAPI/NascarPage/NascarPage/DTOs/LecturaCampeonDTO.cs
using NascarPage.Entitys;
using System.ComponentModel.DataAnnotations;

namespace NascarPage.DTOs
{
    public class LecturaCampeonDTO
    {
        public int Id { get; set; }
        public int Year { get; set; }
        public string AutoCampeon { get; set; } = null!;
        public LecturaPilotoDTO Piloto { get; set; } = null!;
    }
}

File: P6-NascarAngularPage/src/app/componentes/campeones/LecturaCampeonDTO.ts
import { LecturaPilotoDTO } from "../pilotos/LecturaPilotoDTO";

export interface LecturaCampeonDTO {
    id: number;
    year: number;
    autoCampeon: string;
    piloto: LecturaPilotoDTO;
}
export interface PatchCampeonDTO {
    year: number;
    pilotoId: number;
}

export interface CrearCampeonDTO extends PatchCampeonDTO {
    autoCampeon: File | null | undefined;
}